apiserver/test/api_server_service_monitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
This file is a close duplicate of [monitor.yaml](./apiserver/deploy/base/monitor.yaml). I think we should probably use that file.

Reply:
oh, which one? the link goes nowhere

Reply:
https://github.com/blublinsky/kuberay/blob/be4a836a1b7a9f40382d808b27e0c0e17873ec37/apiserver/deploy/base/monitor.yaml#L1

Reply:
Oh the one there is broken, there is no label, that it refers to in the standard deploy

Reply:
I think we should keep deployment related files under the `deployment` directory. That way we can potentially support kustomize based deployments. Perhaps moving these to files to the deployment directory is a better option.

Reply:
Its still there. Just service monitor is removed

Reply:
I still think that we need to move this file and pod monitor under the deployment directory

Reply:
I can go either way. In general Prometheus support is not part of common deployment. It is optional and typically done outside of the standard deployment.